table/init.go

```diff
@@ -74,3 +78,14 @@ func Configure() {
 func SetCsCapacity(capacity int) {
 	csCapacity = capacity
 }
+
+func CreateFIBTable(fibTableAlgorithm string) {
+	switch fibTableAlgorithm {
+	case "hashtable":
+		m := core.GetConfigUint16Default("tables.fib.hashtable.m", 5)
+		newFibStrategyTableHashTable(m)
+	default:
```

Comment:
I think you should put `nametree` here, and let the default branch to `logFatal`.